certification/internal/policy/operator/operator_pkg_name_uniqueness.go

```diff
@@ -35,7 +34,7 @@ type packageData struct {
 type OperatorPkgNameIsUniqueCheck struct{}
 
 func (p *OperatorPkgNameIsUniqueCheck) Validate(bundleRef certification.ImageReference) (bool, error) {
-	annotations, err := containerutils.GetAnnotationsFromBundle(bundleRef.ImageFSPath)
+	annotations, err := getAnnotationsFromBundle(bundleRef.ImageFSPath)
```

Comment:
So these were initially in util-like packages _because_ they could be used regardless of engine. Moving them into this check means only this check can use it. We probably don't have any other case right now where we can reuse these functions, but this organizational move _does_ limit our ability to reuse this. 

Do you think we reconsider collapsing methods like this one into this check (where it's currently used) and instead just consider a more proper name and organization for the package?

Reply:
It's not in the check itself. It's package level. I'm pretty sure it's used in two different checks.

If it does need to be used by, say, a container check, then we might consider moving it to its own package. But for now, it's just an unexported, package-level function defined in operator.go.

The point of a lot of this is that I think we jumped to some of the utils-like stuff too quickly, rather than moving them when we do actually need them in other places.

Reply:
Yup, okay, brain is lagging behind - I read too much into the file name and forgot that those don't really matter.

With that said - I think there's still an opportunity to move this somewhere _else_ where it might make more sense. We can move forward for now and re-organize this when it needs to be done.

Feel free to Resolve Conversation on this point.